docs/user/exams/instructors_guide.rst

```diff
@@ -0,0 +1,192 @@
+
+.. figure:: instructor/add_exercises.png
+   :alt: Add different Exercises
+   :align: center
+
+   Add different Exercises
+
+Registering Students
+^^^^^^^^^^^^^^^^^^^^
+- To register students to the exam, navigate from the exam management to the *Students* page. Here you are presented with two options to register students. You can: 
+
+    1. Add students manually my searching via the search bar
+    2. Bulk import students using a ``.csv`` file. You can do this by pressing the ``Import students`` button.
+
+- You can also remove students from the exam. When you do so, you have the option to also delete their participations and submissions linked to the user’s student exam. 
+    
+    - **Hint:** Just registering the students to the exam will not allow them to participate in the exam. First, individual student exams must be generated. For more information see `Manage Student Exams`_.
+
+.. figure:: instructor/add_students.png
+   :alt: Register Students
+   :align: center
+
+   Register Students Page
+
+Manage Student Exams
+^^^^^^^^^^^^^^^^^^^^
+-  The student exams are managed via the *Student Exams* page. 
+- Here you can have an overview of all student exams. When you press ``View`` on a student exam, you can view the ``details of the student``, the allocated ``working time``, his/her ``participation status``, their ``summary`` as well as their ``scores``. Additionally, you will also be able to view which assessor is responsible for each exercise. 
+
+    - **Hint:** You can change the individual ``working time`` of students from here. 
+
+- To generate the student exams, you must click on ``Generate individual exams``. This will automatically create a student exam for every registered user. The number of exercises will be determined by the flag set in the exam configuration, see `Create and Configure Exam`_. The exercises itself will be randomly selected from the available exercise variants per exercise group, see `Exercise Groups`_.
+- The ``Generate individual exams`` button will be locked once the exam becomes visible to the students. You cannot perform changes to the student exams once the :ref:`exam conduction <exam_conduction>` has started. 
+- If you have added more students recently, you can choose to ``Generate missing individual exams``. 
+- ``Prepare exercise start`` creates a participation for each exercise for every registered user, based on their assigned exercises. It also creates the individual repositories and build plans for programming exercises. This action can take a while if there are many registered students due to the communication between the VC and CI server. 
+
+    - **Warning:** ``Prepare exercise start`` must be executed before the :ref:`exam conduction <exam_conduction>` begins. 
+
+- On the *Student Exams* page, you can also maintain the repositories of the student exams. You can choose to ``lock the repositories`` and ``unlock`` them.
```

Comment:
This needs more explanation. Mention that this only affects programming exercises. Artemis locks and unlocks those automatically based on the individual exam start and end time. The buttons are typically not necessary except something went wrong.